### DISCUSSION TITLE
Minimize .spacedrive files

### DISCUSSION THREAD

**Discussion by yougotwill** [spacedriveapp/spacedrive#1965] [Feature Request]
Hi, firstly really excited about this app and it's development. I was wondering if there was a way to store metadata without having a `.spacedrive` file in each location you add. I have hidden files visible on my mac and the `.spacedrive` is visible on my desktop. Is there any way to centralize this data?

**Comment by utkubakir** [comment]
We plan to rework this and have an option to not use it as it just helps us connect your library with the _external_ drives.
